Source/WebKit/UIProcess/RemoteLayerTree/RemoteScrollingCoordinatorProxy.h

```diff
@@ -97,7 +97,7 @@ class RemoteScrollingCoordinatorProxy : public CanMakeWeakPtr<RemoteScrollingCoo
     virtual void cacheWheelEventScrollingAccelerationCurve(const NativeWebWheelEvent&) { }
     virtual void handleWheelEvent(const WebWheelEvent&, WebCore::RectEdges<bool> rubberBandableEdges);
     void continueWheelEventHandling(const WebWheelEvent&, WebCore::WheelEventHandlingResult);
-    virtual void wheelEventHandlingCompleted(const WebCore::PlatformWheelEvent&, WebCore::ScrollingNodeID, std::optional<WebCore::WheelScrollGestureState>) { }
+    virtual void wheelEventHandlingCompleted(const WebCore::PlatformWheelEvent&, WebCore::ScrollingNodeID, std::optional<WebCore::WheelScrollGestureState>, bool /* wasHandled */) { }
```

Comment:
Can the parameter just have an actual name instead of it being commented out? The style checker is alright with names in headers as long as they are not redundant.

Reply:
It will generate an unused parameter warning because of the empty implementation.

Reply:
Oh true, I didn’t see the implementation and assumed it was just a declaration.